README.md

```diff
@@ -507,6 +508,60 @@ stringData:
   PASSWORD: ********
 ```
 
+###### Additional Information about Customization
+
+You can customize `Secret`'s `metadata` and `stringData` sections.
+
+- In the `metadata`, you can customize only its labels and annotations.
+- You can customize `stringData` or utilize one of the available formatting options as detailed in the [Formats of Service Binding Secrets](#formats-of-service-binding-secrets) section.
+
+**Important**:  If you customize `stringData`, your customization takes precedence over the pre-defined formats (if you parallely provided one of them).
+
```

Comment:
In general, I would change the order...

<table>

You can customize `Secret`'s `metadata` and `stringData` sections.
- In the `metadata`, you can customize only its labels and annotations.

**Important**:  If you customize `stringData`, your customization takes precedence over the pre-defined formats (if you parallely provided one of them).

<sample>
If needed you can utilize one of the available formatting options as detailed in the [Formats of Service Binding Secrets](#formats-of-service-binding-secrets) section, and customize only the metadata section.
E.g., adding labels or annotations to the secret, without changing the pre-defined format of the secret.

Example of using both the pre-defined formats and customization of the Metadata section:

<2nd sample>